app/models/automation_request.rb

```diff
@@ -37,6 +37,13 @@ def self.create_from_ws(version, user, uri_parts, parameters, requester)
     create_request(options, user, auto_approve)
   end
 
+  def self.create_from_scheduled_task(user, uri_parts, parameters)
+    [:namespace, :class_name].each { |key| uri_parts.delete(key) if uri_parts.key?(key) }
+    approval = {'auto_approve' => true}
+    uri_parts.stringify_keys!.reverse_merge!('namespace' => "/System", 'class' => "Process")
```

Comment:
Sorry, @syncrou, didn't see this until now... 

We have this above:
```ruby
   DEFAULT_NAMESPACE = "SYSTEM"
   DEFAULT_CLASS     = "PROCESS"
```

I don't like the hardcoded System and Process here as these are more than just simple strings, they're actual things, the default namespace and class.  

```ruby
uri_parts.stringify_keys!.reverse_merge!('namespace' => "/#{DEFAULT_NAMESPACE.capitalize}", 'class' => DEFAULT_CLASS.capitalize)
```

It's hard to tell if that default hash we're merging in is a THING by itself or if what I did above is good enough.

cc @gmcculloug @bdunne (maybe you have suggestions)